public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -693,6 +696,19 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
         </div>
       );
 
+      const extensionMenuItems = getPanelHeaderExtensionsMenu({ panel, dashboard });
```

Comment:
I don't want to load all plugins just to render a dashboard, can we do this lazy when the menu os opened?